Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -163,19 +163,19 @@ def scan_repositories(repos, person, args)
         tap.install
       end
 
-      repo_full_name = if repo == "brew"
+      nwo = if repo == "brew"
```

Comment:
Sorry, preferred old name! Think `nwo` is pretty opaque to non-GitHubbers. Any of `repo` or `repository` or `repo_name_with_owner` or `repository_name_with_owner` or the old `repo_full_name` will do.

Reply:
Yeah, fair, it's just in my head as "the thing that an owner/name" is - but only because I'm a Hubber.